OCAExplorer/src/App.tsx

```diff
@@ -14,19 +14,25 @@ import {
 import { CssBaseline } from "@mui/material";
 import { StyledEngineProvider, ThemeProvider } from "@mui/material/styles";
 
-const setNewUserCookie = () => {
-    document.cookie = 'OCAExplorerSeenDemo=true; path=/;';
-}
-const seenDemo: string | undefined = document.cookie.split('; ').find((c) => c.split("=")[0] == 'OCAExplorerSeenDemo');
-
 function App() {
   const [overlayData, setOverlayData] = useState<{
     overlay: OverlayBundle | undefined;
     record: CredentialExchangeRecord | undefined;
   }>({ overlay: undefined, record: undefined });
 
-  const cookieValue = seenDemo
-  console.log(cookieValue);
+  // Track if we should skip the rest of the demo.
+  const [demoState, setDemoState] = useState<DemoState>(
+    localStorage.getItem('OCAExplorerSeenDemo') != null

```

Comment:
Can we just store the demo state directly in the `localStorage`?

Reply:
I tried this but the issue is that it would also be writing the state of the times the info button is pressed so it's not ideal since if the info button is pressed then they refresh the page it will think the user has not seen the demo